recipes/physiofit/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "physiofit" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/physiofit-{{ version }}.tar.gz
+  sha256: a0a13d2ef193884d94c0485190d866e661062ef6e1ab6dcae6b2a1e9bcea6f5f
+
+build:
+  entry_points:
+    - physiofit = physiofit.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - poetry-core
+    - pip
+  run:
+    - python >3.9.7
```

Comment:
should that not be `>=3.8` as well?

Reply:
No actually a dependency has problems with python 3.9.7 so I decided to get a greater than version